<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeIP</class>
 <widget class="QDialog" name="ChangeIP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>sure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>NetMask</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>IPAddress</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> </string>
    </property>
    <property name="text">
     <string>IP is dhcp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>450</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Noto Sans&quot;;</string>
    </property>
    <property name="text">
     <string>cancel</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> </string>
    </property>
    <property name="text">
     <string>IP is static</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"> 
 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
